openpype/pipeline/__init__.py

```diff
@@ -93,7 +93,9 @@
     get_current_project_name,
     get_current_asset_name,
     get_current_task_name,
+    get_current_versioning_start
 )
+from .version_start import get_versioning_start
```

Comment:
You have to move this import much earlier. Maybe at the beginning.

Reply:
Still getting a circular import error, with the import after constants.

Reply:
Did you move the `"get_versioning_start"` in `__all__` too?

Reply:
NOTE: Please consider that most of functions inside `openpype.pipeline` should not use global current context (and context tools)... We want to avoid that as much as possible -> not possible all the time, but if it is possible, it's always better. It caused so much trouble in past...